### PR TITLE
Create Movie_Recommendation.py

### PR DESCRIPTION
**Pull request by SahilSatyam**
# Pull Request Template

**Feature Name: Movie Recommendation**

### Self Check(Tick After Making pull Request)

- [x] I have read and followed the [Contribution Guidelines](/CONTRIBUTING.md)
- [x] This issue was assigned to me.
- [x] One Change in one Pull Request
- [x] My file is in the proper folder

- [x] PR is regarding Code
- [x] I am following clean code and Documentation and my code is well linted with flake8.
- [ ] I am using external libraries and I have created requirements.txt in my project folder
- [ ] Script is tested and running perfectly fine on my system

- [ ] PR is regarding the documentation

> If the issue was not assigned to you Please don't make a PR, It will be marked as invalid.

Discuss in discord server:- http://pragmaticprogrammer.in/discord

## acknowledgment
- [x] I have tested my code and distributing it under GPL-3.0 License

### CODE REVIEW

File: Projects/Movie_Recommendation.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    return title
+
+
+# Driver Function
+if __name__ == '__main__':
+
+    print(
+        "Select Your Emotion:\n 1. Anger\n 2. Anticipation\n 3. Disgust\n 4. Fear\n 5. Joy\n 6. Sad\n 7. Surprise\n 8. Trust"
+    )
+    emotion = input("Enter the emotion: ")
```
Comment (Punit-Choudhary) [inline Projects/Movie_Recommendation.py:76]: How about asking for a serial number instead of emotion?

Comment (Punit-Choudhary) [reply Projects/Movie_Recommendation.py:76]: emotion is case sensitive

**Review by Punit-Choudhary — CHANGES_REQUESTED** [review]

**Comment by Punit-Choudhary** [comment]
Please resolve the requested changes and also create a folder with requirements.txt
This program is using external libraries!

File: Projects/Movie_Recommendation.py (hunk 1 above)
Comment (SahilSatyam) [reply Projects/Movie_Recommendation.py:76]: okay i'll do the require changes.

**Review by SahilSatyam — COMMENTED** [review]
done the required changes.

**Review by Punit-Choudhary — APPROVED** [review]
cool, you have made two commits 😄

File: Projects/Movie_Recommendation.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    # IMDb Url for Western genre of
+    # movie against emotion Trust
+    elif (emotion == 8):
+        urlhere = 'http://www.imdb.com/search/title?genres=western&title_type=feature&sort=moviemeter, asc'
+
+    # IMDb Url for Film_noir genre of
+    # movie against emotion Surprise
+    elif (emotion == 7):
+        urlhere = 'http://www.imdb.com/search/title?genres=film_noir&title_type=feature&sort=moviemeter, asc'
+
```
Comment (Punit-Choudhary) [inline Projects/Movie_Recommendation.py:53]: Missing check if user enters wrong input

Comment (Punit-Choudhary) [reply Projects/Movie_Recommendation.py:53]: No need for so many `if - elif` statements.
Make a `dict` store serial number and genres and make URL
Use that URL to make the request.

If the serial number doesn't exist in `dict`, you can add a wrong input warning!

**Review by Punit-Choudhary — CHANGES_REQUESTED** [review]

**Comment by Punit-Choudhary** [comment]
Hey @SahilSatyam 
I really liked this project idea, I can help you build it.
I am reopening it to help you make those changes.
Are you interested or given up?

**Pull request closed without merging**